README.md

```diff
@@ -69,8 +69,20 @@ Considering the data provided above, the application should return the following
 ```
 
 ## Criteria
-You may use any language and framework, provided that you build a solid system with an emphasis on code quality, simplicity, readability, maintainability, and reliability; particularly with regards to architecture and testing. We'd prefer it if you used Python, but it's just that – a preference.
+You may use any language and framework provided that you build a solid system with an emphasis on code quality, simplicity, readability, maintainability, and reliability, particularly regarding architecture and testing. We'd prefer it if you used Python, but it's just that – a preference.
+
+Be aware that Origin will mainly take into consideration the following evaluation criteria:
+* How clean and organized your code is;
+* If you implemented the business rules correctly;
+* How good your automated tests are (qualitative over quantitative).
+
+Other important notes:
+* Develop an extensible recommendation engine, it will be essential to the take-home interview
```

Comment:
I'd drop the spoiler. I would let just "Develop an extensible recommendation engine."

Reply:
About the spoiler, I think you mean `tech on-site` instead of `take-home interview`. Anyway, i'd drop it!